### ISSUE TITLE
markup lists as lists on the article page

### ISSUE THREAD

**Issue by StephDriver** [openlibhums/janeway#4244]
Some parts of articles are presented like lists, but not marked up as lists.  Review whether it would be better to use a programatic list structure for these.  

Consider [WCAG 1.3.1 Info and Relationships](https://www.w3.org/TR/WCAG22/#info-and-relationships)
> Information, structure, and relationships conveyed through presentation can be programmatically determined or are available in text.

and particularly [Technique H48](https://www.w3.org/WAI/WCAG22/Techniques/html/H48.html) in regards to lists.

Areas of concern:
- list of abbreviations, currently being included as comma separated values (NB could mark them up as abbreviations too, as per #4209)
- [x] references, currently being included as a series of paragraphs
- [x] keywords, currently being included as comma separated values
- [x] authors, currently being included as comma separated values (NB not all places were authors are listed are identifying them as authors other than by visual styling - so make sure list is marked up to screenreaders as being of authors) NB only in Material.
-  #4462

**Comment by StephDriver** [comment]
### [misused list example](https://politicalphilosophyjournal.org/article/id/15481/):

a poem that should be written in a blockquote, instead has been marked-up as an unordered list.

<img width="640" alt="image" src="https://github.com/BirkbeckCTP/janeway/assets/5330770/fd9e105e-c191-4ded-a390-fe3a5e770465">


```html
<ul class="list-simple">
<li><p>That Night a Fire did break out—</p></li>
<li><p>You should have heard Matilda Shout!</p></li>
<li><p>You should have heard her Scream and Bawl,</p></li>
<li><p>And throw the window up and call</p></li>
<li><p>To People passing in the Street—</p></li>
<li><p>(The rapidly increasing Heat</p></li>
<li><p>Encouraging her to obtain</p></li>
<li><p>Their confidence)—but all in vain!</p></li>
<li><p>For every time she shouted ‘Fire!’</p></li>
<li><p>They only answered ‘Little Liar!’”</p></li>
</ul>```

**Comment by joemull** [comment]
Refinement: 

We can omit the following item from this issue as it is user-generated content: "lists being misused for non-list information." @StephDriver to make a separate issue for this one.

The rest need to be done for the clean theme.

**Comment by StephDriver** [comment]
> We can omit the following item from this issue as it is user-generated content: "lists being misused for non-list information." @StephDriver to make a separate issue for this one.

I have done this (#4462 ), copied across the example of this behaviour in a comment above and then hidden that comment as off topic.
